docs/openapi/components/schemas/common/PGAStatusMessageCertificate.yml

```diff
@@ -0,0 +1,110 @@
+$linkedData:
+  term: PGAStatusMessageCertificate
+  '@id': https://w3id.org/traceability#PGAStatusMessageCertificate
+title: PGA Status Message Certificate
+description: Certificate Issued by a PGA for the status of Cargo
+type: object
+properties:
+  '@context':
+    type: array
+    const:
+      - 'https://www.w3.org/2018/credentials/v1'
+      - 'https://w3id.org/traceability/v1'
+  type:
+    type: array
+    const:
+      - VerifiableCredential
+      - PGAStatusMessageCertificate
+  id:
+    type: string
+  name:
+    type: string
+  description:
+    type: string
+  issuanceDate:
+    type: string
+  issuer:
+    type: object
+  credentialSubject:
+    title: Credential Subject
+    description: List of PGA Status Messages pertaining to per-line items
+    type: array
+    items:
+      $ref: ./PGAStatusMessage.yml
+  proof:
+    type: object
+  relatedLink:
+    title: Related Link
+    description: Links related to this verifiable credential
+    type: array
+    items:
+      $ref: ./LinkRole.yml
+additionalProperties: false
+required:
+  - type
+example: |-
+  {
+    "@context": [
+      "https://www.w3.org/2018/credentials/v1",
+      "https://w3id.org/traceability/v1"
+    ],
+    "id": "https://example.com/credential/123",
+    "type": [
+      "VerifiableCredential",
+      "PGAStatusMessageCertificate"
+    ],
+    "name": "PGA Hold Certificate",
+    "description": "Certificate Issued by a PGA for the status of Cargo",
+    "relatedLink": [],
+    "issuanceDate": "2019-12-11T03:50:55Z",
+    "issuer": {
+      "id": "did:key:z6MktHQo3fRRohk44dsbE76CuiTpBmyMWq2VVjvV6aBSeE3U",
+      "type": "Organization",
+      "name": "Waters Inc",
```

Comment:
@BenjaminMoe , I recommend you make this issuer more relatable. "Waters Inc" does not sound at all like a PGA. 
Call it something like "Food Border Force Agency" and fbf.example.gov.

Reply:
Thanks for the feedback. Will update with the provided PGA name.